src/ar_http_iface_tests.erl

```diff
@@ -0,0 +1,830 @@
+%%%
```

Comment:
FYI: I have just added a commit to the [style guide PR](https://github.com/ArweaveTeam/arweave/pull/100) recommending to put tests into separate files.

Reply:
Good idea.  A few others here: https://github.com/ArweaveTeam/arweave-private/issues/8